### PR TITLE
docs: add bug triage guidelines

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

**Review by Lightning00Blade — APPROVED** [review]

File: docs/contributing.md [hunk 1]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
 
 2. Create a GitHub release for each package, following the practice of previous
    releases.
+
+## Bug triage guidelines
+
+[Check incoming bug reports](https://github.com/puppeteer/puppeteer/issues) that do not have a `confirmed` or `needs-feedback` label:
+
+1. Make sure the issue labeled as either `bug` or `feature`.
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
1. Make sure the issue is labeled as either `bug` or `feature`.
```

File: docs/contributing.md [hunk 2]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
...
+2. If the issue does not have a clear repro or you cannot repro, ask for the repro and set the `needs-feedback` label.
+3. Follow-up on the issues you previously asked for a feedback on (you should get a notification on GitHub when the user responds).
+4. If the user does not provide feedback, the issue will be closed by the stale bot eventually.
+5. If you are able to reproduce the issue, add the label "confirmed".
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
5. If you are able to reproduce the issue, add the label `confirmed`.
```

File: docs/contributing.md [hunk 3]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
...
+6. If the bug is on the Chromium side, create a corresponding crbug.com issue and label the Github issue with the "upstream" label and post a link to crbug.com in the comments.
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
6. If the bug is on the Chromium side, create a corresponding crbug.com issu, label the GitHub issue with the `upstream` label, and post a link to crbug.com in the comments.
```

File: docs/contributing.md [hunk 4]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
...
+7. If the issue is not related to either Puppeteer or Chromium, close the issue.
+8. If the issue is about missing/incorrect documentation, label it as `documentation`.
+
+Issues with PDFs:
+
+1. If the issue reproduces using the regular print dialog and/or headful, file a crbug.com against the Blink Layout component.
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
1. If the issue reproduces using the regular print dialog and/or headful, [file a crbug.com against the `Blink>Layout` component](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3ELayout).
```

File: docs/contributing.md [hunk 5]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
...
+2. If the issue is specific to the headless headless, file an issue on crbug.com against the headless component.
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
2. If the issue is specific to Headless mode, [file an issue on crbug.com against the `Internals>Headless` component](https://bugs.chromium.org/p/chromium/issues/entry?components=Internals%3EHeadless).
```

**Review by mathiasbynens — APPROVED** [review]
LGTM % suggestions

File: docs/contributing.md [hunk 6]
```diff
@@ -322,3 +322,21 @@ In the event release-please fails, the following needs to be done:
...
+## Bug triage guidelines
+
+[Check incoming bug reports](https://github.com/puppeteer/puppeteer/issues) that do not have a `confirmed` or `needs-feedback` label:
+
+1. Make sure the issue is labeled as either `bug` or `feature`.
+2. If the issue does not have a clear repro or you cannot repro, ask for the repro and set the `needs-feedback` label.
+3. Follow-up on the issues you previously asked for a feedback on (you should get a notification on GitHub when the user responds).
+4. If the user does not provide feedback, the issue will be closed by the stale bot eventually.
+5. If you are able to reproduce the issue, add the label `confirmed`.
+6. If the bug is on the Chromium side, create a corresponding crbug.com issu, label the GitHub issue with the `upstream` label, and post a link to crbug.com in the comments.
```
Comment (mathiasbynens) [inline docs/contributing.md]:
```suggestion
6. If the bug is on the Chromium side, create a corresponding crbug.com issue, label the GitHub issue with the `upstream` label, and post a link to crbug.com in the comments.
```

**Pull request merged**